src/tests/sqlsmith/src/expr.rs

```diff
@@ -88,6 +123,23 @@ impl<'a, R: Rng> SqlGenerator<'a, R> {
         expr.or_else(|| make_general_expr(func.func, exprs))
             .unwrap_or_else(|| self.gen_simple_scalar(ret))
     }
+
+    fn gen_agg(&mut self, ret: DataTypeName) -> Expr {
+        let funcs = match AGG_FUNC_TABLE.get(&ret) {
+            None => return self.gen_simple_scalar(ret),
+            Some(funcs) => funcs,
+        };
+        let func = funcs.choose(&mut self.rng).unwrap();
+        let expr: Vec<Expr> = func
+            .inputs_type
+            .iter()
+            .map(|t| self.gen_expr(*t, true, true))
```

Comment:
avoid magic values.
```
let can_agg = true;
let inside_agg = true;
```

Reply:
Ok understand, will try to change this bad habit haha